packages/core/src/particle/ParticleRenderer.ts

```diff
@@ -464,6 +478,47 @@ export class ParticleRenderer extends GeometryRenderer {
     this._isFadeOut = value;
   }
 
+  /**
+   * Whether play on enable.
+   */
+  get playOnEnable(): boolean {
+    return this._playOnEnable;
+  }
+
+  set playOnEnable(value: boolean) {
+    this._playOnEnable = value;
+
+    if (value) {
+      this.start();
+    }
+    else {
+      this.stop();
+    }
+  }
```

Comment:
Why start/stop() after setting `playOnEnable`. can you use `component.enabled` to replace it?